Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM python:3-alpine
```

Comment:
I don't like the magic this assumes. 3-alpine isn't a tag, but it is found.
https://hub.docker.com/layers/library/python/alpine/images/sha256-fea4a37c69135f7b2a546a55a75e286678a7e2ab1752899a0fd221131a2e856a
Perhaps: python:3.12.3-alpine3.19?

Reply:
It is **your** magic!  
```
$ git blame Dockerfile | head -1
785f799a (Mark Tompsett 2022-08-22 22:34:54 -0400  1) FROM python:3-alpine
```
And I found the `3-alpine` tag:
https://hub.docker.com/layers/library/python/3-alpine/images/sha256-c583b8590a197db1f6efece2dd244b0259cb6f82c4acc654bfbe48e00e20a7b9?context=explore

Reply:
That wasn't the sha that was downloaded when I started it up. It came up with 3.16 as I linked above.

Reply:
The following tags were all pushed 23 days ago:

* 3-alpine
* 3-alpine3.18
* 3-alpine3.19
* 3.12.3-alpine
* 3.12.3-alpine3.18
* 3.12.3-alpine3.19

I assume that means `3-alpine` is picking up Python 3._recent_ running on Alpine 3._recent_ where _recent_ is replaced by whatever the developers believe are currently supportable versions. Since we don't have a reason to be more specific, using `3-alpine` seems a reasonable choice.